Add rustfmt check

wojciechkepka opened the pull request:
Should fix #9. I've added max_width=120 as @lzutao suggested.

lzutao commented on .github/workflows/main.yml:
```diff
@@ -45,3 +45,11 @@ jobs:
       - run: cargo build --all-targets --all
+
+  fmt:
+    runs-on: ubuntu-latest
```
There is fmt build job commented. Please use it instead.

wojciechkepka commented:
> Thanks for the PR.
> What if you creating `.rustfmt.toml` file with the following content:
> 
> ```toml
> use_small_heuristics = "Max"
> merge_derives = false
> max_width = 120
> ```
I've added the config file.

> Also there is already fmt build job in main.yml
Somehow I've missed it. I've removed the new one and uncommented the old one

lzutao commented on .rustfmt.toml:
```diff
@@ -0,0 +1,3 @@
+use_small_heuristics = "Max"
```
Could you add `edition = "2018"`? We have just switched to Rust 2018 edition?

wojciechkepka commented:
I've moved the pull request to #29
